demo-playground/src/main/java/com/google/android/flexbox/FlexItemEditFragment.kt

```diff
@@ -248,8 +248,8 @@ internal class FlexItemEditFragment : DialogFragment() {
                     if (i + 1 < textViews.size) {
                         textViews[i + 1].requestFocus()
                     } else if (i == textViews.size - 1) {
-                        val inputMethodManager = activity
-                                .getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
+                        val inputMethodManager = activity?.getSystemService(
+                                Context.INPUT_METHOD_SERVICE) as InputMethodManager
                         inputMethodManager.hideSoftInputFromWindow(v.windowToken, 0)
```

Comment:
If `activity` is `null`, then `inputMethodManager` is also `null`. But there is no safe-call here.

Reply:
Done